dial_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package forwarder
+
+import (
+	"context"
+	"net/netip"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/saucelabs/forwarder/log/stdlog"
+)
+
+func TestDialerLookupHost(t *testing.T) {
+	if _, ok := os.LookupEnv("CI"); ok {
```

Comment:
We should run in in CI. We have tests talking to google.com so a DNS query to CF won't hurt.